main.c

```diff
@@ -370,11 +372,28 @@ int main(int argc, char **argv)
          case 'j':
              opt_jlinkarm = optarg;
              break;
+         case 'h':
+             opt_help = 1;
+             break;
          default:
              return EXIT_FAILURE;
          }
     }
 
+    if (opt_help) {
+        printf("rtt2pty: Segger RTT to PTY bridge\n");
+        printf("rtt2pty [-opt <param>]\n");
+        printf("\nOptions:\n");
+        printf("\t-d <s>            Segger device ID\n");
+        printf("\t-i <swd:jtag>     Interface\n");
+        printf("\t-s <n>            J-Link serial number\n");
+        printf("\t-S <n>            SWD/JTAG speed\n");
+        printf("\t-b <n>            Buffer\n");
+        printf("\t-2 <0:1>          Bi-dir comms\n");
```

Comment:
`-2` does not have extra parameter, it's just a c&p error in `options` definition

Reply:
I updated all the strings to be a bit clearer in the latest push.